packages/main/src/Calendar.ts

```diff
@@ -264,9 +262,6 @@ class Calendar extends CalendarPart {
 	 * Defines the selected date or dates (depending on the <code>selectionMode</code> property)
 	 * for this calendar as instances of <code>ui5-date</code>.
 	 *
-	 * @type {sap.ui.webc.main.ICalendarDate[]}
-	 * @name sap.ui.webc.main.Calendar.prototype.default
-	 * @slot dates
 	 * @public
 	 */
 	@slot({ type: HTMLElement, invalidateOnChildChange: true, "default": true })
```

Comment:
I think this should be defined in the class comment ? @nnaydenow can you confirm that ?